cdap-docs/reference-manual/source/http-restful-api/metadata.rst

```diff
@@ -556,14 +505,114 @@ Entities that match the specified query and entity type are returned in the body
      - Description
    * - ``namespace-id``
      - Namespace ID
-   * - ``entity-type``
-     - Restricts the search to either all or specified entity types: ``all``, ``artifact``, ``app``, ``dataset``,
-       ``program``, ``stream``, or ``view``
-   * - ``showHidden``
-     - Metadata search hides entities whose name starts with "_" (underscore) from search results. 
-       Set this to ``true`` to include these hidden entities in search results. Default is ``false``.
    * - ``term``
      - :ref:`Query term <http-restful-api-metadata-query-terms>`, as described below. Query terms are case-insensitive.
+   * - ``entity-type``
+     - Restricts the search to either all or specified entity types: ``all``, ``artifact``, ``app``, ``dataset``,
+       ``program``, ``stream``, ``view``
+   * - ``option``
+     - Options for controlling cursors, limits, offsets, the inclusion of hidden entities, and sorting:
+
+       .. list-table::
+          :widths: 20 80
+          :header-rows: 1
+
+          * - Option Name
+            - Option Value, Description, and Notes
+          * - ``cursor``
+            - Cursor to move to in the search results. This would be the value returned in the
+              ``cursor`` field of a response of a previous metadata search request.
+          * - ``limit``
+            - The number of metadata search entities to return in the ``results``. By default, there is no limit.
+          * - ``numCursors``
+            - An upper limit on the number of cursors to return after the ``limit`` parameter is applied to
+              to the search results. Default is ``0``.
+          * - ``offset``
```

Comment:
I think its better for the order of these parameters to be ``query``, ``sort``, ``offset``, ``limit``, ``cursor``, ``numCursors``, ``showHidden``